src/App.tsx

```diff
@@ -1,20 +1,47 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import React from 'react';
+import {
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
 
 import './App.scss';
 
-export const App = () => {
-  return (
-    <div data-cy="app">
-      <Navbar />
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
+import { Navbar } from './components/Navbar/Navbar';
 
-      <div className="section">
-        <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
-        </div>
-      </div>
-    </div>
-  );
-};
+export const App: React.FC = () => (
+  <div data-cy="app">
+    <Navbar />
+
+    <main className="section">
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/people">
+          <Route index element={<PeoplePage />} />
+          <Route path=":selectedPerson" element={<PeoplePage />} />
+        </Route>
+        <Route path="*" element={<NotFoundPage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+      </Routes>
+    </main>
+  </div>
+);
+
+// export const App = () => {
+//   return (
+//     <div data-cy="app">
+//       <Navbar />
+
+//       <div className="section">
+//         <div className="container">
+//           <h1 className="title">Home Page</h1>
+//           <h1 className="title">Page not found</h1>
+//           <PeoplePage />
+//         </div>
+//       </div>
+//     </div>
+//   );
+// };
```

Comment:
Please delete all unusual code, comments and consolelogs before pushing the task on GitHub